web/src/pages/playground.tsx

```diff
@@ -0,0 +1,253 @@
+import { type NextPage } from "next";
+import { useState, useEffect } from "react";
+import TextareaAutosize from 'react-textarea-autosize';
+import Head from "next/head";
+import Link from "next/link";
+
+import { queryLLM, getStampyContent, runSearch } from "../hooks/useSearch";
+import type { Mode, Entry, LLMSettings } from "../types";
+import Header from "../components/header";
+import Chat from '../components/chat';
+import { Controls } from "../components/controls";
+
+const MAX_FOLLOWUPS = 4;
+const DEFAULT_PROMPTS = {
```

Comment:
I considered fetching these from the server, but couldn't be bothered to. It's a potential security risk, but not really that big of one, considering the code is available anyway... The downside of having it here is duplication and so the possibility of things getting out of sync. It's shouldn't be an issue, especially as this is pretty much just for testing